### DISCUSSION TITLE
Exif ifd metadata and basic photo edit

### DISCUSSION THREAD

**Discussion by ippocratis** [Radiokot/photoprism-android-client#32] [Ideas]
Video player already made the app a more standalone image viewer

An option to view photo metadata will make the image browsing flow more consistent as for now someone has to open it with another app to view exif data
To take it ever further being able to watch the photoprism  added keywords for a photo will be dope as then someone looking for similar pictures can search for the apropriate keyword using search

I know this maybe is beyond the scope of the app as it already stated in the readme that features that already are available  in the webapp are not planed to be added.
Maybe an option to open the file in the browser will make sense in this case?
The browser option is  missing from both share and "open in" menus

Performing basic photo actions like rotate,  crop, markup etc will make the app even more a standalone photo/video gallery solution.

Thanks 
keepup

**Comment by Radiokot** [comment]
Hmm, nice idea to open the media in the web client 🤔 Would be a short bridge between the basic app and the full PhotoPrism functionality
